src/com/owncloud/android/ui/activity/FileActivity.java

```diff
@@ -762,6 +768,30 @@ private void onSynchronizeFolderOperationFinish(
             t.show();
         }
     }
+
+    private void onSynchronizeFileOperationFinish(SynchronizeFileOperation operation,
+                                                  RemoteOperationResult result) {
+        dismissLoadingDialog();
+        OCFile syncedFile = operation.getLocalFile();
+        if (!result.isSuccess()) {
+            if (result.getCode() == ResultCode.SYNC_CONFLICT) {
+                Intent i = new Intent(this, ConflictsResolveActivity.class);
+                i.putExtra(ConflictsResolveActivity.EXTRA_FILE, syncedFile);
+                i.putExtra(ConflictsResolveActivity.EXTRA_ACCOUNT, getAccount());
+                startActivity(i);
+
+            }
+
+        } else {
+            if (operation.transferWasRequested()) {
+
```

Comment:
What's the target of this blank block?

I would negate the condition in the ```if``` to apply the code in ```else```directly.